superset/views/core.py

```diff
@@ -1599,12 +1606,13 @@ def publish(  # pylint: disable=no-self-use
 
     @has_access
     @etag_cache(
-        0,
+        CACHE_DEFAULT_TIMEOUT,
```

Comment:
Setting `max_age` will just set cache expiration to a far future date (1 year from now), which is almost never what we want unless we are serving static resources.

Reply:
for Dashboard's etag, it will prefer to be expire in far future, since cache will be validated per user request (with dashboard's last modified_on). `CACHE_DEFAULT_TIMEOUT` may used by the Superset system for other purpose, for example, backend query results cache age, it probably only 1 day or 36 hours.

Reply:
OK, changed it back to `0`.